pkg/app/utils.go

```diff
@@ -21,9 +21,13 @@ import (
 )
 
 const (
-	dbTemplateURI = "https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/%s-persistent-template.json"
+	dbTemplateURI = "https://raw.githubusercontent.com/openshift/origin/v3.11.0/examples/db-templates/%s-%s-template.json"
+	//persistentStorage storage = "persistent"
```

Comment:
should uncomment `persistentStorage`?

Reply:
Hey @PrasadG193,  we should not uncomment `persistentStorage` because we are  not using it anywhere as of now.